lib/msal-browser/FAQ.md

```diff
@@ -1,17 +1,158 @@
+
+**[Authentication](#Authentication)**
+
+1. [I don't understand the redirect flow. How does the handleRedirectPromise function work?](#i-dont-understand-the-redirect-flow-how-does-the-handleredirectpromise-function-work)
+1. [How can I support authentication with personal Microsoft accounts only?](#how-can-i-support-authentication-with-personal-microsoft-accounts-only)
+1. [How do I get an authorization code from the library?](#how-do-i-get-an-authorization-code-from-the-library)
+
+**[Single-Sign-On](#Single-Sign-On)**
+
+1. [How to get single sign-on in my application with MSAL.js?](#how-to-get-single-sign-on-in-my-application-with-msaljs)
+1. [How can my application recognize a user after sign-in? How do I correlate users between applications?](#how-can-my-application-recognize-a-user-after-sign-in-how-do-i-correlate-users-between-applications)
+
+**[Configuration](#Configuration)**
+1. [What is the difference between sessionStorage and localStorage?](#what-is-the-difference-between-sessionstorage-and-localstorage)
+1. [What are the possible configuration options?](#what-are-the-possible-configuration-options)
+1. [Where is the authority string on Azure AD Portal?](#where-is-the-authority-domain-string-on-azure-ad-portal)
+
+**[Tokens](#Tokens)**
+
+1. [How do I acquire an access token? How do I use it?](#how-do-i-acquire-an-access-token-how-do-i-use-it)
+1. [How do I renew tokens with MSAL.js?](#how-do-i-renew-tokens-with-msaljs)
+1. [How can I acquire tokens faster?](#how-can-i-acquire-tokens-faster)
+1. [I'm seeing scopes openid, profile, email, offline_access in my tokens, even though I haven't requested them. What are they?](#im-seeing-scopes-openid-profile-email-offline_access-and-userread-in-my-tokens-even-though-i-havent-requested-them-what-are-they)
+1. [How long do tokens last? How long are they valid for?](#how-long-do-tokens-last-how-long-are-they-valid-for)
+1. [What are the differences between supported audiences and account types?](#what-are-the-differences-between-supported-audiences-and-account-types)
+
+**[Scopes & Resources](Scopes-&-Resources)**
+
+1. [My application has multiple resources it needs to access to. How should I handle scopes for access tokens?](#my-application-has-multiple-resources-it-needs-to-access-to-how-should-i-handle-scopes-for-access-tokens)
+
+
+**Common Issues**
+1. [Why is MSAL throwing an error?](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-browser/docs/errors.md)
+
+***
+
+# Compatibility
+
+## What browsers are supported by MSAL.js?
+
+MSAL.js has been tested with the following browsers:
 
-# Does this library work for iframed applications?
+IE 11, Edge, Chrome, Firefox and Safari
+
+Keep these steps in mind when [using MSAL.js with IE](https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki/Using-msal.js-with-Internet-Explorer).
+
+There are certain known issues and mitigations documented for Safari, IE and Edge. Please check out:
```

Comment:
nit: Very similar to the sentence above.Either move them to the same line or change the language.

Reply:
This was just copied over from what was already in the Wiki. Edits can be made later on a case by case basis.